src/vsr/replica.zig

```diff
@@ -4838,19 +4737,23 @@ pub fn ReplicaType(
                     return;
                 }
 
-                if (message.header.command == .do_view_change) {
-                    const message_log_view = message.header.timestamp;
-                    if (self.log_view_durable() < message_log_view) {
+                // For DVCs and SVCs we must wait for the log_view to be durable:
+                // - A DVC includes the log_view.
+                // - A SV implies the log_view.
+                if (message.header.command == .do_view_change or
+                    message.header.command == .start_view)
+                {
+                    if (self.log_view_durable() < self.log_view) {
```

Comment:
I'd maybe use `message.header.timestamp` rather than `self.log_view` here. THey are he same, `timestamp` is a bit more obscure, _but_ it emphasizesthat we care about data in the message. Though, both are weird tbh, no strong opinion here.

Reply:
Ah, perhaps `const message_log_view = message.header.timestamp;` is the best way to spell this.

Reply:
I slightly prefer how it is right now, since `message.header.timestamp` is only set for `do_view_change`.